core/field_dropdown.ts

```diff
@@ -227,14 +223,9 @@ export class FieldDropdown extends Field {
         this.getSourceBlock().RTL ? FieldDropdown.ARROW_CHAR + ' ' :
                                     ' ' + FieldDropdown.ARROW_CHAR));
     if (this.getSourceBlock().RTL) {
-      // AnyDuringMigration because:  Argument of type 'SVGTSpanElement | null'
-      // is not assignable to parameter of type 'Node'.
-      this.getTextElement().insertBefore(
-          this.arrow_ as AnyDuringMigration, this.textContent_);
+      this.getTextElement().insertBefore(this.arrow_, this.textContent_);
```

Comment:
`this.arrow_` apparently changed at some point since being given `AnyDuringMigration` since it worked when I just removed `AnyDuringMigration`.